nexus/types/src/external_api/params.rs

```diff
@@ -1406,12 +1409,12 @@ pub enum DiskSource {
 /// Create-time parameters for a `Disk`
 #[derive(Clone, Debug, Deserialize, Serialize, JsonSchema)]
 pub struct DiskCreate {
-    /// common identifying metadata
+    /// The common identifying metadata for the disk.
     #[serde(flatten)]
     pub identity: IdentityMetadataCreateParams,
-    /// initial source for this disk
+    /// The initial source for this disk.
     pub disk_source: DiskSource,
-    /// total size of the Disk in bytes
+    /// The total size of the Disk (in bytes).
```

Comment:
we're not super consistent about this, but I prefer to avoid periods on things that are not sentences

Reply:
Yeah, we're totally inconsistent within `omicron/nexus/types/src/external_api/params.rs`. I'll try doing a more complete review separately.